modules/index/src/main/java/org/apache/ignite/internal/index/IndexManager.java

```diff
@@ -181,7 +149,7 @@ public void stop() throws Exception {
      *      parameters.
      */
     CompletableFuture<MvTableStorage> getMvTableStorage(long causalityToken, int tableId) {
-        return tableStoragesVv.get(causalityToken).thenApply(ignore -> tableStoragesById.get(tableId));
+        return tableManager.tableAsync(causalityToken, tableId).thenApply(table -> table.internalTable().storage());
```

Comment:
I believe that this method no longer belongs to `IndexManager` and you could remove it, but if that's an unrelated change then we can do it later. What's you opinion?